ckanext/spark/templates/snippets/organization.html

```
@@ -0,0 +1,39 @@
+      {% block heading %}
+      <h5 class="org-heading">ORGANIZATION</h5>
+	<div class=“org-title”>{{ organization.title or organization.name }}
+        {% if organization.state == 'deleted' %}
+          [{{ _('Deleted') }}]
+        {% endif %}
+      </div>     
+
+      {% endblock %}
+      {% block description %}
+      {% if organization.description %}
+        <p class="description">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p class="description"> ] on line 18.

(tag-pair)